src/marine/godae_profile2ioda.py

```diff
@@ -254,11 +259,17 @@ def main():
         obsList.append(obs)
 
     varDict = {
-        'ob_tmp': 'sea_water_temperature',
-        'ob_sal': 'sea_water_salinity'
+        #              var name,             units
+        'ob_tmp': ['sea_water_temperature', '???'],
+        'ob_sal': ['sea_water_salinity', '???']
```

Comment:
Can someone please tell me what the units values should be for these variables? Thanks!

Reply:
Probably `K` and `PSU`, respectively, but I'm not a marine expert and will defer to others.

Reply:
Thanks!